src/seedu/addressbook/commands/Hello.java

```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.commands;
+
+
+public class Hello extends Command {
+
+    public static final String COMMAND_WORD = "hello";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": prints hello World!";
+
+
+
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code.